### PR TITLE
connectors: added docs for SQL Server (Debezium) connector

### PR DESCRIPTION
**Pull request by bochenekmartin**
## Description

Resolves: https://redpandadata.atlassian.net/browse/CIAINFRA-452

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+| Format of the value in the Redpanda topic.
+
+| `Message value JSON contains schema`
+| `value.converter.schemas.enable`
+| Enable to specify that the message value contains schema in the schema field.
+
+
+| `Max tasks`
+| `tasks.max`
+| Specifies the maximum number of tasks that the connector can use to capture data from the database instance. If the `Databases` list contains more than one element, you can increase the value of this property to a number less than or equal to the number of elements in the list. Default: 1
```
Comment (tomasz-sadura) [inline modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc:83]: It clashes with the `SQL Server (Debezium) connector can work with only a single task at a time.` statement from Limitations section.

**Review by tomasz-sadura — APPROVED** [review]

File: modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
```
Comment (micheleRP) [inline modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc:83]:
```suggestion
| The maximum number of tasks that the connector can use to capture data from the database instance. If the `Databases` list contains more than one element, you can increase the value of this property to a number less than or equal to the number of elements in the list. Default: 1
```

File: modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+:page-categories: Deployment, Integration
+
+You can use an SQL Server (Debezium) Source connector to import updates to Redpanda from SQL Server.
+
+== Prerequisites
+
+Before you can create an SQL Server (Debezium) Source connector in the Redpanda Cloud, you
+must:
+
+* Make the SQL Server (Debezium) database accessible from connectors instance.
```
Comment (micheleRP) [inline modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc:13]:
```suggestion
* Make the SQL Server (Debezium) database accessible from the connector instance.
```

File: modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+| Globally-unique name to use for this connector.
+|===
+
+. Click *Next*. Review the connector properties specified, then click *Create*.
+
+== Map data
+
+Use the appropriate key or value converter (input data format) for your data as follows:
+
+* Use `Include Schemas`, `Include Tables` and `Include Columns` properties to define
```
Comment (micheleRP) [inline modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc:96]:
```suggestion
* Use the `Include Schemas`, `Include Tables`, and `Include Columns` properties to define
```

File: modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+* Use only `JSON` (`org.apache.kafka.connect.json.JsonConverter`), `AVRO` (`io.confluent.connect.avro.AvroConverter`),
+and `CloudEvents` (`io.debezium.converters.CloudEventsConverter`) formats for the Kafka message key and value format.
+
+== Test the connection
+
+After the connector is created:
+
+. Open Redpanda Console, click the *Topics* tab, and select a topic.
+Check to confirm that it contains data migrated from SQL Server.
+Alternatively, use the `rpk consume` to check the topic.
```
Comment (micheleRP) [inline modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc:109]:
```suggestion
Alternatively, run `rpk consume` to check the topic.
```

File: modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+and `CloudEvents` (`io.debezium.converters.CloudEventsConverter`) formats for the Kafka message key and value format.
+
+== Test the connection
+
+After the connector is created:
+
+. Open Redpanda Console, click the *Topics* tab, and select a topic.
+Check to confirm that it contains data migrated from SQL Server.
+Alternatively, run `rpk consume` to check the topic.
+. Click the *Connectors* tab to confirm no issues have been reported for the
```
Comment (micheleRP) [inline modules/develop/pages/managed-connectors/create-sqlserver-connector.adoc:110]:
```suggestion
. Click the *Connectors* tab to confirm that no issues have been reported for the
```

**Pull request merged**